specs/refactor_common_code_and_move.adoc

```diff
@@ -0,0 +1,206 @@
+= Refactor modules like tendrl-gluster-integartion, tendrl-node-agent etc and
```

Comment:
Line breaks do not work in titles, if I'm not mistaken. Shorten the main title. Add a subtitle if you wish to convey additional context.